addons/waypoint/README.md

```diff
@@ -0,0 +1,8 @@
+## Waypoint
+
+Waypoints:
+- Vehicle Move - Order vehicle to move in strain line to target ignoring AI orders and situation. waypoint create pool of vehicles where group units are driver and create new group per vehicle
```

Comment:
You should keep one empty line before and after any list in Markdown. Plus some spelling fixes.

I'm not sure what that does mean, so if my suggestion doesn't apply, please fix that.
> Waypoint creates pool of vehicles where group units are driver and create new group per vehicle
```suggestion
## Waypoints

New Waypoints:

- Vehicle Move - Orders vehicles to move in straight line to the waypoint position, ignoring AI orders and situation. Splits the original group to one group per vehicle for the duration of the move.
```